work/about.html

```diff
@@ -0,0 +1,132 @@
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<head>
+    <title>About me - Marie-Pier Dubois</title>
+    <link rel="stylesheet" href="../css/bootstrap.min.css">
+    <link rel="stylesheet" href="../css/styles.css">
+    <link rel="stylesheet" href="https://use.typekit.net/uqv7xml.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg bg-navbar">
+        <div class="container-fluid">
+            <a class="navbar-brand active" href="../index.html"><img src="../assets/mpd-logo5-mauvebleu.svg" alt="Logo" class="mpd-logo"></a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link" href="../index.html">Home</a>
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                            Work
+                        </a>
+                        <ul class="dropdown-menu">
+                            <li><a class="dropdown-item" href="branding.html">Branding</a></li>
+                            <li><a class="dropdown-item" href="websites.html">Websites</a></li>
+                            <li><a class="dropdown-item" href="lettering.html">Lettering</a></li>
+                        </ul>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link active" aria-current="page" href="about.html">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="contact.html">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    
+    <div class="container mt-3">
+        <h1>About</h1>
+    </div>
+
+    <section class="sections-margins">
+      <div class="card mb-3 noborders-card" style="max-width: 1000px">
+          <div class="row g-0">
+            <div class="col-md-4">
+              <img src="../assets/mpd_picture.jpeg" class="img-fluid rounded-start about-img-home" alt="This is me">
+            </div>
+            <div class="col-md-8">
+              <div class="card-body">
+                <p class="card-text">Hi there!
+                <p>I'm Marie-Pier and I'm a Canadian illustrator and graphic designer based in Quebec who specializes in lettering.</p>
```

Comment:
Here you have a `<p>` open and I could not find where it is closing. Try to format your code using Visual Studio Code to check any indentation issues.